src/server/views/lists/partials/translators-interpreters/translators-interpreters-results-list.njk

```diff
@@ -18,8 +18,14 @@
     <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
     <strong class="govuk-warning-text__text">
       <span class="govuk-warning-text__assistive">Warning</span>
-      Translators and interpreters must be authorised to operate in some countries (known as sworn translators).
-      Check providers are qualified to carry out the service you need in the country you have selected.
+      {% if hasSworn %}
```

Comment:
According to the spec in trello, I this condition should be inverted

Reply:
Really good spot. This should be fixed now even-though Github hasn't triggered it.